patches/@lerna+package-graph+4.0.0.patch

```diff
@@ -0,0 +1,114 @@
+diff --git a/node_modules/@lerna/package-graph/index.js b/node_modules/@lerna/package-graph/index.js
```

Comment:
Do we need to change this filename if the lerna is upgraded? How can we auto-detect and/or autofix?

What if the minor version changes as part of 'npm upgrade'?

Reply:
I mean the file name - `@lerna+package-graph+4.0.0.patch`, not the filename in the line.

Reply:
`patch-package` will fail if there's a version mismatch or if it cannot apply the patch.

Reply:
Should we also prevent upgrades? https://github.com/aws/aws-cdk/blob/81b0ef609835778d5122a32d2e91e7881da84a41/.github/workflows/yarn-upgrade.yml#L49

This would also be a nice place to put a link to the upstream PR and a comment about what we're doing here, and that it should be removed when the upstream PR is merged and released. (posterity)